symbols/planet_vndr/cosmo

```diff
@@ -1236,3 +1236,55 @@ xkb_symbols "hu" {
 
     key <AB08>  { [     comma,   question, XF86Option,   multiply ] };
 };
+
+default  partial alphanumeric_keys
+xkb_symbols "pl" {
+    include "planet_vndr/cosmo(modifiers)"
+    include "planet_vndr/cosmo(common_keys)"
+
+    name[Group1]="Gemini Polish";
```

Comment:
Cosmo?

Reply:
All the other are 'Gemini Language' (guess that's the result of copying it from gemini?) so kept it the same, but can change to 'Cosmo Polish', will submit another one with the XF86 keys mapped too once that is supported

Reply:
Updated to 'Cosmo Polish'

Reply:
I've not been paying too much attention lately, I've added you to the project so you can pop it through if you like. I'm more inclined towards adding people to the project and doing reviews before builds. Nikita seems to want do do things differently this time around with regards to the special keys.